### PR TITLE
feat: Implemented new speakers view 

### PR DESCRIPTION
**Pull request by vvyn**

### CODE REVIEW

**Review by raywa04 — DISMISSED** [review]
lgtm!

File: styles/globals.css [hunk 1]
```diff
@@ -1,4 +1,8 @@
 @import url('/fonts/stylesheet.css');
+@import url('https://fonts.googleapis.com/css2?family=Fredoka:wght@300..700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@300...700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300...700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300..700&display=swap');
```
Comment (doanhtu07) [inline styles/globals.css]: In my PR #261, I think I also download some fonts there, including DM Sans, Fredoka One, and Nunito. Maybe we can figure out how to merge my code with yours later on.

**Review by doanhtu07 — DISMISSED** [review]
LGTM! 

But we will need to figure out merging with my fonts. 

And also, don't forget to fix the merge conflicts with branch `develop`. 

Everything else looks good.

File: styles/globals.css (hunk 1 above)
Comment (vvyn) [reply styles/globals.css]: Yeah it shouldn't be too bad. We'll see based on who's code gets merged first.

**Review by raywa04 — DISMISSED** [review]

**Pull request merged**